src/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
- [ ] Kindly consider removing the trailing slash because it has no effect on void elements and interacts badly with unquoted attribute values.
```suggestion
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```